src/app/technical-record/vehicle-summary/vehicle-summary.component.html

```diff
@@ -60,130 +63,123 @@
-            <div *ngIf="axel.parkingBrakeMrk">
+          <div *ngIf="axel.parkingBrakeMrk">
               <span id="{{ 'test-parkingBrakeMrkAxle-' + axel.axleNumber }}"
                 >Axle {{ axel.axleNumber }}</span
               >
-            </div>
           </div>
         </div>
-      </div>
+      </ng-container>
     </dd>
     <dd class="govuk-summary-list__value"></dd>
   </div>
-  <div class="govuk-summary-list__row" *ngIf="activeRecord.vehicleType == 'trl'">
+  <div class="govuk-summary-list__row" *ngIf="activeRecord.vehicleType === 'trl'">
     <dt class="govuk-summary-list__key">
-      Load sensing valve
+      Suspension type
     </dt>
-    <dd id="test-loadSensingValve" class="govuk-summary-list__value">
-      <div *ngIf="!!activeRecord.brakes; else hyphen">
-        {{ activeRecord.brakes.loadSensingValve | DefaultNullOrEmpty }}
-      </div>
+    <dd id="test-suspensionType" class="govuk-summary-list__value">
+      {{ activeRecord.suspensionType | DefaultNullOrEmpty }}
     </dd>
     <dd class="govuk-summary-list__value"></dd>
   </div>
-  <div class="govuk-summary-list__row" *ngIf="activeRecord.vehicleType == 'trl'">
+  <div class="govuk-summary-list__row">
     <dt class="govuk-summary-list__key">
-      Antilock braking system
+      Road friendly suspension
     </dt>
-    <dd id="test-antilockBrakingSystem" class="govuk-summary-list__value">
-      <div *ngIf="!(activeRecord.brakes == null); else hyphen">
-        {{ activeRecord.brakes.antilockBrakingSystem | DefaultNullOrEmpty }}
-      </div>
+    <dd id="test-roadFriendly" class="govuk-summary-list__value">
+      {{ activeRecord.roadFriendly | DefaultNullOrEmpty }}
     </dd>
     <dd class="govuk-summary-list__value"></dd>
   </div>
-  <div class="govuk-summary-list__row" *ngIf="activeRecord.vehicleType == 'trl'">
+  <div class="govuk-summary-list__row">
     <dt class="govuk-summary-list__key">
-      Brake actuator
+      Vehicle class
     </dt>
-    <dd id="test-brakeActuator" class="govuk-summary-list__value">
-      {{ activeRecord.axles.slice(-1)[0].brakes.brakeActuator | DefaultNullOrEmpty }}
+    <dd id="test-vehicleClassDescription" class="govuk-summary-list__value">
+      {{
+        vehicleClassDescription
+      }}
     </dd>
     <dd class="govuk-summary-list__value"></dd>
   </div>
-  <div class="govuk-summary-list__row" *ngIf="activeRecord.vehicleType == 'trl'">
+  <div class="govuk-summary-list__row">
     <dt class="govuk-summary-list__key">
-      Lever length
+      Vehicle configuration
     </dt>
-    <dd id="test-leverLength" class="govuk-summary-list__value">
-      <div *ngIf="activeRecord.axles[1] != null; else hyphen">
-        <span>{{ activeRecord.axles[1].brakes.leverLength | DefaultNullOrEmpty }}</span>
-      </div>
+    <dd id="test-vehicleConfiguration" class="govuk-summary-list__value">
+      {{
+        (activeRecord.vehicleConfiguration | DefaultNullOrEmpty).charAt(0).toUpperCase() +
```

Comment:
I thought we had an util function for this in 10166? Is that possible to use it please?